lisa/sut_orchestrator/azure/features.py

```diff
@@ -596,6 +600,103 @@ def get_raw_data_disks(self) -> List[str]:
             disk_array[int(disk.split("/")[-1].replace("lun", ""))] = cmd_result.stdout
         return disk_array
 
+    def add_managed_data_disk(
+        self,
+        names: List[str],
+        luns: List[str],
+        size: str = "20",
+        location: str = "westus2",
+    ) -> None:
+        platform: AzurePlatform = self._platform  # type: ignore
+        compute_client = get_compute_client(platform)
+
+        # create managed disk
+        managed_disks = []
+        for name in names:
+            async_disk_update = compute_client.disks.begin_create_or_update(
+                self._resource_group_name,
+                name,
+                {
+                    "location": location,
+                    "disk_size_gb": size,
+                    "creation_data": {"create_option": DiskCreateOption.empty},
+                },
+            )
+            managed_disks.append(async_disk_update.result())
```

Comment:
should we have separate cases for different disk type sku?

Reply:
I think this makes sense. I will add them separately for each disk type